doc/specification.md

```diff
@@ -115,6 +115,10 @@ Timestamping operations.
 * Validation of next-hops
 * resolution outside of gRIBI
 
-
-
-
+## About gRIBI server caching
+gRIBI server implementation is not required to cache all installed objects.
+Implications:
+  * When a VRF is removed (e.g. accidentally by user via cli):
+    * The device is not required to maintain gRIBI objects in the FIB or RIB.
+    * Get() or Flush() should return failed (because the VRF is no longer there)
```

Comment:
Can this be clarified a bit more either here, or in https://github.com/openconfig/gribi/pull/49/files wrt to Get() returning error. Is the expectation that
a) if Get() is issued on a VRF that does not exist in the system (ie not configure) the Get() RPC be errored ?
b) if Get() is issued on a VRF that exists in the system, but does not have the requested AFTType objects programmed thru GRIBI, then a empty GetResponse is returned, and RPC is not errored.

Reply:
@ashpatcisco your understanding is correct. 

a. yes
b. yes

I'll try to include those clarifications as we are filling the spec :) Thanks!